src/js/view/components/LogViewer.js

```diff
@@ -103,14 +158,98 @@ const LogViewer = props => {
     });
   }, [props.source.path]);
 
+  useEffect(() => {
+    const wheelHandler = event => {
```

Comment:
What is a wheel 🤔

Reply:
![image](https://user-images.githubusercontent.com/3668673/74947945-e4dff280-53fb-11ea-95d0-8feae5e9c871.png)

Pierre and his ideas... (It's actually a mousewheel)

But yeah, naming could probably be clearer if that's what you're getting at!

Reply:
How does 'handleScrollWheelEvent' sound to you? Is it still too vague?